src/Go/SaveLoad.ts

```diff
@@ -77,6 +78,18 @@ export function loadGo(data: unknown): boolean {
   Go.currentGame = currentGame;
   Go.previousGame = previousGame;
   Go.stats = stats;
+
+  // If it's the AI's turn, initiate their turn, which will populate nextTurn
+  if (currentGame.previousPlayer === GoColor.black && currentGame.ai !== GoOpponent.none) makeAIMove(currentGame);
```

Comment:
IMO all-on-one-line style for if statements is a bit shady (it's lead to various high-profile bugs in other applications), but especially if you have an else block you should always use curly braces for the if block.